google-style.yml

```diff
@@ -35,13 +35,18 @@ Metrics/PerceivedComplexity:
   Max: 10
 Naming/VariableNumber:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: braces_for_chaining
+Style/EmptyMethod:
+  EnforcedStyle: expanded
 Style/FrozenStringLiteralComment:
   Enabled: false
 Style/MethodCallWithArgsParentheses:
   Enabled: true
   EnforcedStyle: omit_parentheses
   AllowParenthesesInMultilineCall: true
   AllowParenthesesInCamelCaseMethod: true
+  AllowParenthesesInChaining: true
```

Comment:
This is the setting that looks to allow `datastore.query("Visit").order("timestamp", :desc).limit(10)`.